### DISCUSSION TITLE
Slinger Player 'Remote Control' pops up as blank white box

### DISCUSSION THREAD

**Discussion by tryin100** [GerryDazoo/SlingerPlayer#30] [Q&A]
When I use Slinger Player locally on LAN it works great.  Both Player and the 'remote control' (to change channels etc.)
When I use Slinger Player on my windows PC for remote viewing  (i.e. when not on the LAN where the Slingbox and server are located) the 'remote control' just appears as a blank white box.  
I can still use a different device (e.g. my phone) to access and use the 'remote control'.
Any help on how to get the remote control working on my windows PC would be appreciated.

**Comment by GerryDazoo** [comment]
Give this a try
https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section

**Reply by bradut** [reply]
I also had this issue, and Gerry helped me to address it.
For additional info please see [this thread](https://github.com/GerryDazoo/SlingerPlayer/discussions/19),
